com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDEditorUtils.cs

```diff
@@ -282,6 +282,26 @@ internal static void HandlePrefixLabelWithIndent(Rect totalPosition, Rect labelP
         internal static bool IsPresetEditor(UnityEditor.Editor editor)
         {
             return (int)((editor.target as Component).gameObject.hideFlags) == 93;
+        }?
+        internal static void QualitySettingsHelpBox(string message, MessageType type, HDRenderPipelineUI.Expandable uiSection, string propertyPath)
```

Comment:
actually after all I think it's better that way, because that function is just a helper to expand the quality settings foldouts, so we indeed want it to be tied to the type of expandables supported by the quality settings